src/event.rs

```diff
@@ -175,28 +175,31 @@ pub enum WindowEvent {
     /// The window has been destroyed.
     Destroyed,
 
-    /// A file is being hovered over the window.
-    ///
-    /// When the user hovers multiple files at once, this event will be emitted for each file
-    /// separately.
-    HoveredFile(PathBuf),
-
-    /// A file has been dropped into the window.
-    ///
-    /// When the user drops multiple files at once, this event will be emitted for each file
-    /// separately.
-    ///
-    /// The support for this is known to be incomplete, see [#720] for more
-    /// information.
-    ///
-    /// [#720]: https://github.com/rust-windowing/winit/issues/720
-    DroppedFile(PathBuf),
-
-    /// A file was hovered, but has exited the window.
-    ///
-    /// There will be a single `HoveredFileCancelled` event triggered even if multiple files were
-    /// hovered.
-    HoveredFileCancelled,
+    /// A drag operation has entered the window.
+    DragEnter {
+        /// List of paths that are being dragged onto the window.
+        paths: Vec<PathBuf>,
+        /// Position of the drag operation. May be negative on some platforms if something is
+        /// dragged over a window's decorations (title bar, frame, etc).
+        position: PhysicalPosition<f64>,
+    },
+    /// A drag operation is moving over the window.
+    DragOver {
+        /// Position of the drag operation. May be negative on some platforms if something is
+        /// dragged over a window's decorations (title bar, frame, etc).
+        position: PhysicalPosition<f64>,
+    },
+    /// The drag operation has dropped file(s) on the window.
+    DragDrop {
```

Comment:
```suggestion
    DragMoved {
```

so it looks like pointer.

Reply:
Done